lib/cryptoexchange/exchanges/wcx/services/market.rb

```diff
@@ -0,0 +1,69 @@
+module Cryptoexchange::Exchanges
+  module Wcx
+    module Services
+      class Market < Cryptoexchange::Services::Market
+        class << self
+          def supports_individual_ticker_query?
+            true
+          end
+        end
+
+        def fetch(market_pair)
+          ticker_output = super(ticker_url(market_pair))
+          candle_output = super(candles_url(market_pair))
+          adapt(ticker_output, candle_output, market_pair)
+        end
+
+        def ticker_url(market_pair)
+          base = market_pair.base
+          target = market_pair.target
+          "#{Cryptoexchange::Exchanges::Wcx::Market::API_URL}/ticker/#{base}-#{target}"
+        end
+
+        def candles_url(market_pair)
+          base = market_pair.base
+          target = market_pair.target
+          "#{Cryptoexchange::Exchanges::Wcx::Market::API_URL}/candles/#{base}-#{target}/1D"
```

Comment:
@Swingcloud explore using `240` interval instead of `1D`
and sum the first 6 blocks, which is `6 * 4` = 24 hours for better accuracy i believe

Reply:
okay on it

Reply:
seems just volume need to sum up? high and low don't need this